test/forms/changeActivity/SecondaryWarehouseDetailsFormProviderSpec.scala

```diff
@@ -21,10 +21,12 @@ import play.api.data.FormError
 
 class SecondaryWarehouseDetailsFormProviderSpec extends BooleanFieldBehaviours {
 
-  val requiredKey = "changeActivity.secondaryWarehouseDetails.error.required"
+  val requiredKey = "changeActivity.secondaryWarehouseDetails.error.requiredNoWarehouses"
+  val requiredKeyForWarehouses = "changeActivity.secondaryWarehouseDetails.error.required"
   val invalidKey = "error.boolean"
 
-  val form = new SecondaryWarehouseDetailsFormProvider()()
+  val form = new SecondaryWarehouseDetailsFormProvider()(false)
```

Comment:
Perhaps use named boolean parameters here? Don't mind if you don't want to change it